packages/api/src/schema/mutation.ts

```diff
@@ -852,9 +852,17 @@ export default [
       """
       signPaymentProviderForCheckout(orderPaymentId: ID, transactionContext: JSON): String!
 
-      prepareProductMediaUpload(mediaName: String!, productId: ID!): MediaUploadTicket!
-      prepareAssortmentMediaUpload(mediaName: String!, assortmentId: ID!): MediaUploadTicket!
-      prepareUserAvatarUpload(mediaName: String!, userId: ID): MediaUploadTicket!
+      prepareProductMediaUpload(
+        mediaName: String!
+        productId: ID!
+        asPrivate: Boolean
```

Comment:
Looking at the other queries and mutations, we have never used 'as' or 'is' prefixing for the boolean parameters, so just go for private: Boolean here.

Reply:
I see, yeah no if possible we should stick with a single word that describes the purpose if somehow possible, the as,is,set,get things is somehow alien to the current schema, what about **exclusive** ?

Maybe something like that?